content/en/mobile/_index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Datadog Oncall Mobile App
+kind: documentation
+beta: true
+further_reading:
+- link: "/monitors/"
+  tag: "Documentation"
+  text: "Alerting"
+- link: "/dashboards/"
+  tag: "Documentation"
+  text: "Dashboards"
+---
+
+<div class="alert alert-warning"> This service is in public beta. To join the beta, follow the installation instructions. The beta is not currently available to customers using SAML authentication. If you have any feedback, please <a href="/help">contact Datadog support</a>.</div>monitors
+
+The Datadog Oncall app enables on-call engineers to view alerts from Datadog on their mobile device. When notified to a Datadog alert by Pagerduty, Slack or other pager apps, you'll be able to investigate issues by opening monitor graphs and dashboards on your mobile device.
+
+## Installing
+
+The Datadog Oncall App is currently in beta, so it's not available generally in app stores. To download the app, use the specific invite links in these instructions.
+
+### iOS
+
+1. Install the [Testflight app][1] from the App Store.
+2. Open this invite link from your iPhone: [https://apple.co/2xEC5ke][2]. Allow the link to open in the Testflight app. This takes you to a page where you can download the Datadog app.
+3. Once the app downloads, open the app, select whether you are logging into the US or EU site, and log in with your Datadog credentials.
+
+### Android
+
+1. To install the android app, open this invite link from your android: [https://play.google.com/apps/testing/com.datadog.app][3]. This takes you to a page where you can download the Datadog app.
+2. Once the app downloads, open the app, select whether you are logging into the US or EU site, and log in with your Datadog credentials.
+
+### Logging in
+
+You can login using standard and Google authentication. [SAML login][4] is not currently supported. If you have SAML enabled but not strictly, you can access the mobile app by creating a new standard email account. An org admin needs to approve the newly created account.
+
+Switch organizations and log out from the **Account page**.
```

Comment:
Not grasping this line. To log in, you need to log in, switch orgs, and log out from the account page? Or are these instructions for the org admin portion?

Reply:
yeah this was supposed to be removed good catch. 
```suggestion
```